backend/project_alpha/api/tests/test_models.py

```diff
@@ -0,0 +1,51 @@
+from rest_framework.test import APITestCase
+from django.core.exceptions import ValidationError
+from project_alpha.web.models import User
+
+
+class UserModelTestCase(APITestCase):
+
+    def setUp(self):
+        self.username = 'test@example.com'
+        self.user = User.objects.create_user(self.username)
+
+    def test_happy_patch_all_validators_are_not_called(self):
```

Comment:
`test_happy_patch_all_validators_are_not_called`
Thing is that validators were called, but they aren't raised an errors.